index.html

```diff
@@ -17,10 +17,11 @@
         <nav class="nav-bar">
             <a href="#" class="nav-name">Mark Fenekayas</a>
             <ul class="nav-list">
-                <li>Portfolio</li>
-                <li>About</li>
-                <li>Contact</li>
-                <li><i class="fa-regular fa-envelope"></i></li>
+                <span class="nav-list-remove"><i class="fa-solid fa-xmark"></i></span>
```

Comment:
- [ ] On mobile when the menu is open I have realized that it does not fit properly at the screen as it is supposed to. Kindly fix its position so that it matches the provided Figma design.

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/208629462-00196799-e13f-422e-8e25-22303346ed18.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/208629558-2e00af5b-1624-4716-841e-0c3b2e0ed9fb.png)